tests/Unit/Classes/DnsRecordPtr.tests.ps1

```diff
@@ -484,22 +484,21 @@ Describe 'Test bad inputs (both IPv4 and IPv6)' -Tag 'Test', 'DnsRecord', 'DnsRe
                 Ensure    = 'Present'
             }
 
-            { $malformattedIPv4State.Get() } | Should -Throw -ExpectedMessage ('*' + '"Cannot convert value "{0}" to type "System.Net.IPAddress". Error: "An invalid IP address was specified.""' -f $malformattedIPv4State.IpAddress)
+            { $malformattedIPv4State.Get() } | Should -Throw -ErrorId 'MethodArgumentConversionInvalidCastArgument'
```

Comment:
This does not seem correct to me, it looks like the resource doesn't throw an error using the correct method (probably wrongly uses `throw`) so we get an generic error instead of the actual DSC resource localized error. 

Can we get a better error by using the DSC community stamdard:
https://github.com/dsccommunity/SqlServerDsc/blob/1b3ea84057ba7dcbed5eca13d5a4ecde81068126/source/Classes/020.SqlPermission.ps1#L366-L371